sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeContentOperation.cs

```diff
@@ -12,22 +12,62 @@
 
 namespace Azure.AI.FormRecognizer.Models
 {
-    internal class ExtractLayoutOperation : Operation<IReadOnlyList<ExtractedLayoutPage>>
+    /// <summary>
+    /// Tracks the status of a long-running operation for recognizing layout elements from forms.
+    /// </summary>
+    public class RecognizeContentOperation : Operation<IReadOnlyList<ExtractedLayoutPage>>
     {
+        /// <summary>Provides communication with the Form Recognizer Azure Cognitive Service through its REST API.</summary>
+        private readonly ServiceClient _serviceClient;
+
+        /// <summary>The last HTTP response received from the server. <c>null</c> until the first response is received.</summary>
         private Response _response;
+
+        /// <summary>The result of the long-running operation. <c>null</c> until result is received on status update.</summary>
         private IReadOnlyList<ExtractedLayoutPage> _value;
-        private bool _hasCompleted;
 
-        private readonly ServiceClient _operations;
+        /// <summary><c>true</c> if the long-running operation has completed. Otherwise, <c>false</c>.</summary>
+        private bool _hasCompleted;
 
+        /// <inheritdoc/>
         public override string Id { get; }
 
+        /// <inheritdoc/>
         public override IReadOnlyList<ExtractedLayoutPage> Value => OperationHelpers.GetValue(ref _value);
 
+        /// <inheritdoc/>
         public override bool HasCompleted => _hasCompleted;
 
+        /// <inheritdoc/>
         public override bool HasValue => _value != null;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RecognizeContentOperation"/> class.
+        /// </summary>
+        /// <param name="operationId">The ID of this operation.</param>
+        /// <param name="client">The client used to check for completion.</param>
+        public RecognizeContentOperation(string operationId, FormRecognizerClient client)
```

Comment:
In the original PR there is a `cancellationToken` here. although it was not used, it will be good to add it here to the TODO so eventually we come back and add it + its respective usage

Reply:
I thought it would not be good to have it doing nothing in code. Adding it to the TODO is a good idea.

Reply:
For all we know we might not even needed it :P